### ISSUE TITLE
Cannot pass list to --multidir

### ISSUE THREAD

**Issue by Clownshift** [bio-phys/complexespp#32]
After #31 I tried to run the REMC tutorial, but I cannot figure out how to pass the`--multidir` argument.
It is a little challenging to debug, since if **anything** is wrong, the parse just shows the help and does not necessarily tell you **what** is wrong.
I think that `--multidir` has to be used with an equal sign `--multidir=...`, different than all other arguments, for which the `=` can also be omitted. If I omit the `=` for multidir, I simply get the help string.

Up to know, I did not figure out a way how to use multidir, because I believe the expansion of wildcard expressions isn't processed. `complexes++ --config config --multidir=simulation-temp-*`, yields: `File "simulation-temp-*/config" does not exist`.
even though `ls simulation-temp-*/config`. Does yield: 
`simulation-temp-298/config  simulation-temp-318/config  simulation-temp-338/config  simulation-temp-358/config  simulation-temp-378/config  simulation-temp-398/config`

Also I cannot explicitly give a list of directories, at least I would not know in which syntax.

**Comment by Clownshift** [comment]
I did not want to make a separate issue for this, but it is a little inconsistent that we use a hyphen in the argument `--nb-threads`, since in #31 all other arguments use underscores now...

**Comment by berenger-eu** [comment]
Ok so the help should say:
[HELP] Command line with all args: ./src/complexes++ --help --version **--multidir [List of Strings]** --config=[String] --backup=[Boolean] --rerun=[Boolean] --restart=[String] --replex=[Integer] --replex_stat=[Integer] --replex_accept=[String] --movestats=[String] --nb-threads=[Integer] --replex_verbosity=[String]

So you should try
complexes++ --config config --multidir simulation-temp-*

**Comment by Clownshift** [comment]
If I execute `complexes++ --config config --multidir simulation-temp-*` 
I only get the help message...

**Comment by Clownshift** [comment]
I created a separate issue for the messages of the parser (#33 ), which is probably not that urgent to solve. 
But this issue here is not about the messages. I really tried a lot and I did not figure out any way to pass on a list to `--multidir`.

I only meant to give as much info as possible, by telling you that if I do not use `=` with `--multidir` it does not accept the argument at all, apparently.

**Comment by Clownshift** [comment]
Executing `complexes++ --config config --multidir="simulation-temp-*"` still yields:
```
terminate called after throwing an instance of 'std::invalid_argument'
  what():  File "simulation-temp-*/config" does not exist.
```
Also, I did not figure out a way to explicitly give the list, which users should be able to do. None of the things below work:
* `complexes++ --config config --multidir="simulation-temp-298 simulation-temp-318"`
* `complexes++ --config config --multidir simulation-temp-298/,simulation-temp-318`
* `complexes++ --config config --multidir=simulation-temp-298/,simulation-temp-318`
* `complexes++ --config config --multidir=simulation-temp-298/:simulation-temp-318`
* `complexes++ --config config --multidir=simulation-temp-298 simulation-temp-318`
* `complexes++ --config config --multidir=( simulation-temp-298 simulation-temp-318 )`
* `complexes++ --config config --multidir simulation-temp-298 simulation-temp-318 `
* `complexes++ --config config --multidir="simulation-temp-298 simulation-temp-3018"`
* `complexes++ --config config --multidir=simulation-temp-298 simulation-temp-318`
* `complexes++ --config config --multidir simulation-temp-298 simulation-temp-318`
* `complexes++ --config config --multidir (simulation-temp-298 simulation-temp-318)`
* `complexes++ --config config --multidir=(simulation-temp-298 simulation-temp-318)`
* `complexes++ --config config --multidir=simulation-temp-298,simulation-temp-318`
* `complexes++ --config config --multidir=simulation-temp-298:simulation-temp-318`
* `complexes++ --config config --multidir=(simulation-temp-298,simulation-temp-308)`
* `complexes++ --config config --multidir= simulation-temp-298,simulation-temp-308`

**Comment by berenger-eu** [comment]
Several of them should work.
In the previous version:
`complexes++ --config config --multidir simulation-temp-298 simulation-temp-318`
and now also (on branch #34 )
`complexes++ --config config --multidir=simulation-temp-298 simulation-temp-318`
(the errors from the arguments should also be more visible)

**Comment by Clownshift** [comment]
> Several of them should work. In the previous version: `complexes++ --config config --multidir simulation-temp-298 simulation-temp-318` and now also (on branch #34 ) `complexes++ --config config --multidir=simulation-temp-298 simulation-temp-318` (the errors from the arguments should also be more visible)

So, for me neither of the two actually work. Not on main, neither on #34 (see discussion there).
@berenger-eu Does it work for you?

**Comment by Clownshift** [comment]
Minimum example:

https://syncandshare.lrz.de/getlink/fiSxvytKFZaVCzo8bo8FghLP/multidir.tar.gz
